js/plugins/helperLines.js

```diff
@@ -2,11 +2,11 @@
  * Displays the x-coordinate i.e the label and y-coordiante i.e the day
  * when clicked on a timeline element in advanced view.
  */
-function initHelperLines(enableHelperLines, zoomFactor, advancedView, divId, formatTime, daysToTimeObject) {
+function initHelperLines(options) {
```

Comment:
This is not extending some base `Plugin` prototype. Check out inheritance: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Inheritance_and_the_prototype_chain. The idea would be to create some base prototype called Plugin, which the other plugins extend.